requirements-dev.txt

```diff
@@ -1,66 +1,180 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
We use Python 3.8 in the `Jenkinsfile`. This is asking for trouble.

Reply:
I'm using Ubuntu 20.04 on my workstation, and there's no Python 3.8 package available, since ti was replaced with Python 3.8. So I used my jumphost to replace c14b119 with e31e437980886f8943394f0fb46ef968140673d0.

Reply:
Great!
We should add a `.python-version` file to make sure Dependabot uses Python 3.8 as well.